tmail-backend/apps/distributed/src/test/resources/blob.properties

```diff
@@ -0,0 +1,86 @@
+# ========================================= ObjectStorage deduplication ========================================
```

Comment:
I think this file can be removed

Reply:
If we don't have it, DistributedServerTest/EncryptedDistributedServerTest will throw exception